setup.cfg

```diff
@@ -9,6 +9,7 @@ long_description_content_type = text/markdown
 url = https://github.com/miguelgrinberg/flask-httpauth
 project_urls =
     Bug Tracker = https://github.com/miguelgrinberg/flask-httpauth/issues
+    Documentation = http://pythonhosted.org/Flask-HTTPAuth
```

Comment:
The current documentation link is https://flask-httpauth.readthedocs.io/. Would you please update the README.md file with it? Thanks!

Reply:
Done!